bedrock/products/templatetags/misc.py

```diff
@@ -103,6 +131,10 @@ def vpn_subscribe_link(
 
     product_url = f"{settings.VPN_SUBSCRIPTION_URL}subscriptions/products/{product_id}?plan={plan_id}"
 
+    if "analytics" in selected_plan:
+        class_name += " ga-begin-checkout"
+        optional_attributes["data-ga-item"] = _vpn_get_ga_data(selected_plan)
```

Comment:
Issue: I think this blows up if `optional_attributes` are not present. We might need to ensure they exist before making an assignment here?

```
'NoneType' object does not support item assignment
```

Reply:
✅